docs/application/dotnet/guides/device/attached-devices.md

```diff
@@ -215,18 +216,18 @@ To control LEDs on the device:
 
 To lock and unlock the CPU state:
 
--   Lock the power state with the `RequestCpuLock()` method of the [Tizen.System.Power](https://developer.tizen.org/dev-guide/csapi/api/Tizen.System.Power.html) class.
+-   Lock the power state with the `RequestLock()` method of the [Tizen.System.Power](https://developer.tizen.org/dev-guide/csapi/api/Tizen.System.Power.html) class.
```

Comment:
Question:  Why is the method name changed to `RequestLock()`?
As seen in the https://github.com/Samsung/TizenFX/blob/master/src/Tizen.System/Device/Power.cs link `RequestCpuLock()` method name is used.

Reply:
RequestCpuLock and ReleaseCpuLock is deprecated and I made RequestLock and ReleaseLock because there are other locks besides cpu